Makefile

```diff
@@ -27,3 +27,7 @@ run:
 build:
 	rm -rf ./dist
 	hugo -s site --verbose --destination ../dist/html --printUnusedTemplates --printPathWarnings --gc
+
+.PHONY: test
+test:
+	docker run -v $(PWD)/dist/html/:/mnt 18fgsa/html-proofer mnt --ignore_empty_alt
```

Comment:
We should not need to use docker or any other container to run local testing against the docs. Please add a `Makefile` target to run html-proofer without using a container or docker.

Reply:
@neflyte I have updated the Makefile. Please review and suggest if any changes are needed.